### PR TITLE
Fix for compile error when using llvm-3.9.0 

### PR DESCRIPTION
**Pull request by thierry-tct**

### CODE REVIEW

**Comment by thierry-tct** [comment]
Fix for compile error when using llvm-3.9.0

File: tools/llvm-slicer.cpp [hunk 1]
```diff
@@ -958,7 +958,7 @@ static uint32_t parseAnnotationOpt(const std::string& opt)
 
 int main(int argc, char *argv[])
 {
-    llvm::sys::PrintStackTraceOnErrorSignal();
+    llvm::sys::PrintStackTraceOnErrorSignal(llvm::StringRef());
```
Comment (mchalupa) [inline tools/llvm-slicer.cpp]: This needs compilation guards, since in LLVM 3.8 the signature is different (there it takes a bool optionally)

**Comment by mchalupa** [comment]
Hi,

please clean up the commits a bit. Squeeze 68e015d into the previous and get rid of all changes that does not target llvm-slicer.cpp or PostDominators.cpp. Also the llvm-slice.cpp change needs work (commented in the code).

Thanks,
Marek

**Comment by mchalupa** [comment]
fixed in #116

**Pull request closed without merging**